src/3d/qgscameracontroller.h

```diff
@@ -22,6 +22,8 @@
 #include <QRect>
 #include <Qt3DCore/QEntity>
 #include <Qt3DInput/QMouseEvent>
+#include <QImage>
+#include <QMutex>
```

Comment:
is this needed?

Reply:
QImage is needed, QMutex is a leftover from refactoring.